src/utils/ExtensionLoader.js

```diff
@@ -1,24 +1,24 @@
 /*
```

Comment:
Get rid of these changes (e.g., `git checkout bramble src/utils/ExtensionLoader.js`)